Tests/BootCamp.Chapter.Tests/DeckTest.cs

```diff
@@ -32,11 +39,29 @@ public void DrawFromTop_Given_List_Return_Last_Item(Deck deck, Card expected)
             Assert.Equal(expected, actual);
         }
 
-        // DrawRandom cannot be tested because it needed a mock and we did not learn that on lesson1.
+        [Fact]
+        public void DrawRandom_Given_List_One_Card_Returns_That_Card()
```

Comment:
Again you bring the list back. I don't know what is the list you a referring to, talk to me via test as if you are talking to a customer, person, wife, daughter. 

If you tell them- "there is a list in the deck"- they will look at you strange. There are cards in the deck!

In this case *_Given_One_Card*